src/util.h

```diff
@@ -80,8 +81,25 @@ static const int64_t CENT = 1000000;
 void MilliSleep(int64_t n);
 
 extern int GetDayOfYear(int64_t timestamp);
-extern std::map<std::string, std::string> mapArgs;
-extern std::map<std::string, std::vector<std::string> > mapMultiArgs;
+
+/**
+ * Allows search of mapArgs and mapMultiArgs in a case insensitive way
+ */
+
+struct mapArgscomp
+{
+   bool operator() (const std::string& lhs, const std::string& rhs) const
+   {
+       return strcasecmp(lhs.c_str(), rhs.c_str()) < 0;
+   }
+};
+
+typedef std::map<std::string, std::string, mapArgscomp> tArgs;
+typedef std::map<std::string, std::vector<std::string>, mapArgscomp> tMultiArgs;
```

Comment:
I wonder if we should just use std::multi_map here, though it might not be a drop-in replacement.

Reply:
Let's do that in another PR.